### PR TITLE
Add Snapshot tests

### PR DESCRIPTION
**Pull request by GustavoMA**
#### What is the purpose of this pull request?

As described in the title.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: node_js
+cache:
+  directories:
+    - 'node_modules'
+node_js:
+  - '10'
+before_install:
+  - npm i -g yarn
```
Comment (JuanAlmeida12) [inline .travis.yml]: You dont need install yarn.

Comment (JuanAlmeida12) [reply .travis.yml]: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#using-yarn

Comment (GustavoMA) [reply .travis.yml]: Nice, didn't know that.

File: react/__mocks__/vtex.render-runtime.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+)
+
+export const Link = ({ page, className, children }) => (
+  <a href={page} className={className}>
+    {children}
+  </a>
+)
+
+export const withRuntimeContext = comp => comp
+export const withSession = () => comp => comp
```
Comment (theomoura) [inline react/__mocks__/vtex.render-runtime.js]: comp => comp, won't do?

Comment (GustavoMA) [reply react/__mocks__/vtex.render-runtime.js]: Unfortunately no, It is used like this in the code:
```javascript
const content = withSession()(compose(
  injectIntl,
  graphql(LOGIN_OPTIONS_QUERY),
  graphql(session, options),
)(LoginContent))
```

**Pull request merged**